services/app-api/prisma/schema.prisma

```diff
@@ -43,21 +43,22 @@ model State {
 }
 
 model User {
-  id               String             @id
-  givenName        String
-  familyName       String
-  email            String
-  role             Role
-  stateAssignments State[]
-  stateCode        String?
-  questions        Question[]
-  responses        QuestionResponse[]
+  id                 String             @id
+  givenName          String
+  familyName         String
+  email              String
+  role               Role
+  divisionAssignment Division?
+  stateAssignments   State[]
```

Comment:
Would it be useful to have a `updatedAt` field for this `User` table since we can now edit it?

Reply:
great idea